core/test/variable_operations_test.cpp

```diff
@@ -12,6 +12,43 @@
 using namespace scipp;
 using namespace scipp::core;
 
+template <typename T>
+class VariableScalarOperatorTest : public ::testing::Test {
+public:
+  Variable variable;
+  T scalar;
+
+  T value() const { return this->variable.values<T>()[0]; }
+
+  virtual void SetUp() {
+    this->variable = makeVariable<T>({Dim::X, 1}, {10});
+    this->scalar = static_cast<T>(2);
```

Comment:
Doesn't normal member init work here, `Variable variable{makeVariable....}`?

Reply:
Yes, but `variable` is written by the test.
IIRC the class stays alive for the entire test fixture, hence why this is in `SetUp`.

Reply:
IIRC it does *not* with `gtest`. I think it makes a new one for every test?

Reply:
Yes, it is new for each test.
Maybe I was thinking of CxxTest?